_data/banking.yml

```diff
@@ -127,6 +127,13 @@ websites:
       img: bbt.png
       tfa: No
 
+    - name: BBVA Compass
+      url: https://www.bbvacompass.com/
+      img: bbvacompass.png
+      tfa: No
+      facebook: bbvacompass
+      twitter: bbvacompass
```

Comment:
Please capitalize the twitter tag so that it matches their twitter handle. `BBVACompass`

Thank you!